Fashionz/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+    <title>Fashionz</title>
+    <meta name="viewport" content="width=device-width", initial-scale="1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="assets/styles.css">
+</html>
+
+<body class="body-bg-color">
+    <!----------START :: Header section--------->
+    <header>
+        <div class="fashionz-logo">
+            <div><img src="assets/images/logo.png" alt=""></div>
```

Comment:
1. We can start the path with `./ ` for paths : <img src="./assets/images .....
2. You need specify alt value which would help as fallback when the image fails to load
3. do we really a div to wrap the img tag, as it already has parent div right ?

Reply:
Review comments resolved